snmp/datadog_checks/snmp/data/profiles/cisco-catalyst.yaml

```diff
@@ -0,0 +1,410 @@
+# Profile for Cisco Catalyst devices
+
+extends:
+  - _base.yaml
+  - _cisco-generic.yaml
+  
+sysobjectid:
+  - 1.3.6.1.4.1.9.1.111
```

Comment:
Some of these sysobjectids like this one are end-of-life devices (https://www.cisco.com/c/en/us/obsolete/switches/cisco-catalyst-3500-xl-series-switches.html). Keeping those may make the profile hard to read